src/runtime/Util/OpsHelper.cs

```diff
@@ -82,7 +82,7 @@ internal static class EnumOps<T> where T : Enum
     {
         [ForbidPythonThreads]
 #pragma warning disable IDE1006 // Naming Styles - must match Python
-        public static PyInt __int__(T value)
+        public static PyInt __index__(T value)
```

Comment:
Shouldn't `__index__` supplement `__int__` and not completely replace it?

Reply:
According to the docs, `__index__` should be used for lossless conversions, `int(...)` will use it if available.

Reply:
Are you sure there are no common scenarios where only `__int__` will be looked up?

Reply:
No valid scenarios. /We/ do that, but only for our own types :). The correct way to handle things is `operator.index` or `builtins.int`, depending on the expected semantics.

Reply:
Still not sure what is the point of removing `__int__` here. Is this not a valid implementation of `__int__`? Is there some kind of instruction saying `__int__` should not be set in addition to `__index__`?

Reply:
E.g. https://docs.python.org/3/c-api/long.html#c.PyLong_AsLong, functions like these will not work anymore with `__int__`. `__int__` is for lossy conversions, this one is lossless => `__index__` is correct.

Reply:
Compromise: I'll add both, is that okay with you?